build.gradle

```diff
@@ -141,7 +141,9 @@ tasks.withType(Jar){
         attributes 'Main-Class': 'picard.cmdline.PicardCommandLine',
                 'Implementation-Title': 'Picard',
                 'Implementation-Vendor': 'Broad Institute',
-                'Implementation-Version': version
+                'htsjdk-Version': htsjdkVersion,
```

Comment:
This is unnecessary for the fix but seems like a fine addition...

Reply:
Yes.  I was inspired by your example.